api_specification.yaml

```diff
@@ -52,3 +70,17 @@ definitions:
     #       type: string
     #     content:
     #       type: string
+
+  List:
+    type: array
+    # items:
```

Comment:
You also have to explicitly defined what the array contains. See the converted code for examples. 
If you're not sure how to do this, don't worry, I'll handle it